backend/main.py

```diff
@@ -0,0 +1,359 @@
+from flask import Flask, request, jsonify
+from pymongo import MongoClient
+from flask_cors import CORS
+from flask_bcrypt import Bcrypt
+from flask_jwt_extended import JWTManager, create_access_token,jwt_required, get_jwt_identity
+from datetime import datetime, timedelta
+import secrets,pytz,jwt
+
+bcrypt = Bcrypt()
+app = Flask(__name__)
+CORS(app)
+
+
+from twilio.rest import Client
+import random
+account_sid = 'AC3a53ca2d743cda7bfbb3f1e22fb193fb'

```

Comment:
Do not push private data specific to an account to public repo, instead put placeholders in the code before pushing it to git.